README.md

```diff
@@ -27,8 +27,56 @@ Enhanced Failure Monitoring (EFM) is a feature available from the [Host Monitori
 Please visit [this page](./docs/using-the-jdbc-driver/UsingTheJdbcDriver.md#using-the-aws-jdbc-driver-with-plain-rds-databases) for more information.
 
 ## Getting Started
-For more information on how to obtain the AWS JDBC Driver, minimum requirements to use it, and how to integrate the AWS JDBC Driver into your project, please visit the [Getting Started page](./docs/GettingStarted.md).
+For more information on how to obtain the AWS JDBC Driver, minimum requirements to use it, 
+and how to integrate the AWS JDBC Driver into your project, please visit the 
+[Getting Started page](./docs/GettingStarted.md).
+### Maven Central
+You can search on The Central Repository with GroupId and ArtifactId [software.amazon:aws-advanced-jdbc-wrapper][mvn-search].
```

Comment:
NIT: You can find our driver by using the search in The Central Repository using the GroupId and/or ArtifactId [software.amazon:aws-advanced-jdbc-wrapper][mvn-search].